data/announcements/scheduled.yaml

```
@@ -103,3 +103,16 @@ announcements:
   message: |
     2 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
     December 9 + 10, 2021.
+
+- name: Kubecon 2022 EU
+  startTime: 2022-05-09T00:00:00  #Added in https://github.com/kubernetes/website/pull/33562
```

```suggestion
  startTime: 2022-05-10T00:00:00  #Added in https://github.com/kubernetes/website/pull/33562
```
It's 5/9, maybe start 5/10

---

```suggestion
  startTime: 2022-05-10T00:00:00  #Added in https://github.com/kubernetes/website/pull/33578
```